integration_test/Cargo.toml

```diff
@@ -12,6 +12,8 @@ odbc-api = { version = "0.45.1", features = ["narrow"] }
 odbc = "0.17.0"
 lazy_static = "1"
 odbc-sys = { version = "0.21.2", default-features = false }
+mongo-odbc-core = { path = "../core" }
```

Comment:
[nit] I don't think you need to import this anymore

Reply:
Good point!